<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HgStripDialog</class>
 <widget class="QDialog" name="HgStripDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Strip Changesets</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="revGroupBox">
     <property name="title">
      <string>&amp;Revision</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="numberButton">
        <property name="toolTip">
         <string>Select to specify a revision by number</string>
        </property>
        <property name="text">
         <string>Number</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="numberSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter a revision number</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight</set>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="idButton">
        <property name="toolTip">
         <string>Select to specify a revision by changeset id</string>
        </property>
        <property name="text">
         <string>Id:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="idEdit">
        <property name="toolTip">
         <string>Enter a changeset id</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="tagButton">
        <property name="toolTip">
         <string>Select to specify a revision by a tag</string>
        </property>
        <property name="text">
         <string>Tag:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="tagCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter a tag name</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="branchButton">
        <property name="toolTip">
         <string>Select to specify a revision by a branch</string>
        </property>
        <property name="text">
         <string>Branch:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="branchCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter a branch name</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="expressionButton">
        <property name="text">
         <string>Expression:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="expressionEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter a revision set expression</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Bookmark:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="bookmarkCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Enter a bookmark name</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QCheckBox" name="forceCheckBox">
       <property name="toolTip">
        <string>Select to enforce removal of changesets</string>
       </property>
       <property name="text">
        <string>Enforce Operation</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="noBackupCheckBox">
       <property name="toolTip">
        <string>Select to not create backup bundles</string>
       </property>
       <property name="text">
        <string>No Backup</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="keepCheckBox">
       <property name="toolTip">
        <string>Select to keep the current working directory state</string>
       </property>
       <property name="text">
        <string>Keep Working Directory</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>numberButton</tabstop>
  <tabstop>numberSpinBox</tabstop>
  <tabstop>idButton</tabstop>
  <tabstop>idEdit</tabstop>
  <tabstop>tagButton</tabstop>
  <tabstop>tagCombo</tabstop>
  <tabstop>branchButton</tabstop>
  <tabstop>branchCombo</tabstop>
  <tabstop>expressionButton</tabstop>
  <tabstop>expressionEdit</tabstop>
  <tabstop>bookmarkCombo</tabstop>
  <tabstop>forceCheckBox</tabstop>
  <tabstop>noBackupCheckBox</tabstop>
  <tabstop>keepCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HgStripDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HgStripDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numberButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>numberSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>idButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>idEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tagButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>tagCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>branchButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>branchCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>expressionButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>expressionEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
